api/auth/william.go

```diff
@@ -0,0 +1,96 @@
+package auth
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+	"github.com/topfreegames/extensions/pg/interfaces"
+	"github.com/topfreegames/maestro/errors"
+	"github.com/topfreegames/maestro/models"
+	"github.com/topfreegames/maestro/william"
+	"net/http"
+	"strings"
+)
+
+type PermissionResolver interface {
```

Comment:
Shouldn't this interface and the "default" implementation live at `api/auth/auth.go` or something since it is no specific to `william`?

Reply:
The concept of different permissions are only relevant to william.